tools/deploy_translatable_strings.sh

```diff
@@ -14,74 +14,94 @@
 
 # Script for pushing the translatable messages to poBranch.
 
+# DO NOT `set -x`.  We have to pass secrets to `openssl` on the command line,
+# and we don't want them appearing in the log.  This script instead manually
+# 'echo's its status at various points.
+set -eu -o pipefail
+
+if [[ "$#" -ne 1 ]]; then
+    echo "Usage: deploy_translatable_string.sh /path/to/translations/artifact" >&2
+    exit 1
+fi
 
 # Variables used by this script.
 # From github actions the docs/locale/en directory from the sphinx build
 # gets downloaded from the github actions artifacts as deploy directory
-TARGET_REPOSITORY="git@github.com:qiskit-community/qiskit-translations.git"
-SOURCE_DIR=`pwd`
-SOURCE_LANG='en'
-
-SOURCE_REPOSITORY="git@github.com:Qiskit/qiskit.git"
-TARGET_BRANCH_PO="main"
-DOC_DIR_PO="deploy"
-TARGET_DOCS_DIR_PO="docs/locale"
```

Comment:
This relative directories are meant to refer to different roots, and more than that, the `DOC_DIR_PO` is just a subcomponent of a path, where the full absolute path is supposed to be the root of the translations built artifact.  This is organised by the outer CI workflow now, so I just have it feed in the correct path to this script.